src/showcase-merge/app/shared/component-viewer/one-variant-only-notification/one-variant-only-notification.component.html

```diff
@@ -0,0 +1,14 @@
+<sbb-notification
+  *ngIf="variantOnly"
+  type="info"
+  class="sbb-one-variant-only-notification"
+  readonly
+>
+  <span class="sbb-one-variant-only-notification-title">{{ variantOnly }}-only component</span>
+  <p>
```

Comment:
Should we maybe use textexpand to show/hide the details?

Reply:
textexpand component is not usable inside a notification because a color is defined and would have to be overwritten and "show more" for me is not the best term . Tooltip is not the best because there is already an info symbol and it looks somehow bad. Other option is to create own own textexpand like component or as a small improvement, make info text 13px